00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,542 @@
+discussion. If there are guests to the call, the team should welcome them and
+allow them to ask questions. If there are any other topics to cover, they should
+be brought up here.
+
+Standup should take no longer than a half-hour, and should be scheduled as
+follows:
+
+- 10-15 min: triage
+- 15-20 min: Board review/project discussion (however long is left after triage)
+
+### Retrospective
+
+The final meeting is **Retrospective**, which represents the end of a
+sprint. It is a time for the whole team to reflect on work accomplished, demo
+new things, and discuss how to approach tasks that didn't get completed.
+
+Most of these topics will take the form of discussion. The only significant
+process to detail here is the workflow for reviewing issues and PRs from the
+last sprint, which is what should happen first.
+
+![Sprint Retro diagram](./sprint-retro.svg)
+
+Any issues that don't have an associated PR are assumed to be "not started".
+However, any issue that made it into the sprint in the first place is probably
+still important, so it should be considered as such for the next sprint as well.
+Thus, it will be sent to the next Planning meeting as a Proposed issue (see
+[Planning](#planning) for more details).
+
+If it _does_ have an associated PR, the next question to ask is whether that
+associated PR was merged or not. If it was, then the issue is completed. If not,
+that means the PR is still in review and likely just needs a bit more review
+time. Thus, it should stay as-is in the Under Review column.
+
+Beyond that, the rest of the meeting should be dedicated to demos or other
+high-level discussion about work accomplished.
+
+Retrospectives should take a full hour. As detailed in [Planning](#planning),
+the two meetings should be combined, with Retrospective happening first,
+replacing the standup of the first day of a new sprint.
+
+Here's a proposed schedule:
+
+- 20 min: review issues/PRs from last sprint
+- 40 min: demos or other discussion
+
+## Pull Requests
+
+The meetings above detail how we organize and plan our work with issues. This
+section will elaborate how the team should execute that work with pull requests.
+
+As described [above](#tracking-work), pull requests encapsulate the end of a
+task. Pushing them across the finish line, whether that means providing review
+or addressing those reviews, should always be a priority. As such, it's
+important to communicate who needs to take action on a PR at any given time. The
+Enarx project's current workflow does not do this well, and it's an obvious area
+for improvement.
+
+### Workflow
+
+Here's a flowchart of the new and improved proposed workflow for a pull request,
+starting with a PR's submission and going all the way to its merge:
+
+![PR workflow diagram](./pr-workflow.svg)
+
+As with every issue, every PR will be reviewed through a Triage, during which
+reviewers will be requested and assigned. At least two reviewers must be
```

Comment:
Does this mean that if a PR comes in and you feel motivated to review it, you should *not* review it until the next day's triage? Or does it mean that you can review it and become and "optional reviewer", and other reviewers may be assigned the next day? The latter makes more sense to me.

Reply:
Yes, the latter.

The implication for someone submitting a "changes requested" review is that they're on board to keep reviewing the PR until their feedback is resolved. That doesn't need to happen in a formal triage process. That's mostly there to ensure that PRs _have_ the necessary reviewers.